src/main/java/ac/grim/grimac/commands/GrimHistory.java

```diff
@@ -0,0 +1,84 @@
+package ac.grim.grimac.commands;
+
+import ac.grim.grimac.GrimAPI;
+import ac.grim.grimac.manager.log.Violation;
+import ac.grim.grimac.manager.log.ViolationDatabaseManager;
+import ac.grim.grimac.utils.anticheat.MessageUtil;
+import co.aikar.commands.BaseCommand;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Optional;
+import co.aikar.commands.annotation.Subcommand;
+import io.github.retrooper.packetevents.util.folia.FoliaScheduler;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.command.CommandSender;
+
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@CommandAlias("grim|grimac")
+public class GrimHistory extends BaseCommand {
+
+    @Subcommand("log|logs")
+    @CommandPermission("grim.history")
+    @CommandAlias("gh")
+    public void onLogs(CommandSender sender, OfflinePlayer target, @Optional Integer page) {
+        String header = GrimAPI.INSTANCE.getConfigManager().getConfig().getStringElse("grim-history-header",
+                "%prefix% §bShowing logs for §f%player% (§f%page%§b/§f%maxPages%§b)");
```

Comment:
```suggestion
                "%prefix% §bShowing logs for §f%player% (§f%page%§b/§f%maxPages%§f)");
```
I wonder if all white would look better for the page indicator